bundles/org.openhab.binding.lgwebos/README.md

```diff
@@ -300,7 +282,7 @@ Parameters:
 
 | Name    | Description                                                                   |
 |---------|-------------------------------------------------------------------------------|
-| appId   | The Application ID. getApplications provides available apps and their appIds. |
+| appId   | The Application ID. Console command smarthome:lgwebos <thingUID> applications provides available apps and their appIds. |
```

Comment:
Please forget smarthome prefix, it is not required and I imagine this is something that should change in OH3.

Reply:
sure